FAIL
Kernel panic in run: PANIC at ../../threads/synch.c:241 in lock_release(): assertion `lock_held_by_current_thread (lock)' failed.
Call stack: 0xc002fec8
Translation of call stack:
0xc002fec8: (unknown)
